### PR TITLE
Release notes for 3.0.5

### PR DESCRIPTION
**Pull request by erikmarr**
## Purpose of this pull request

This pull request (PR) updates the release notes for Product Recommendations 3.0.5.

## Affected DevDocs pages

-  https://devdocs.magento.com/recommendations/release-notes.html

### CODE REVIEW

File: src/recommendations/release-notes.md [hunk 1]
```diff
@@ -14,6 +14,11 @@ The release notes include:
 -  {:.new}New features
 -  {:.fix}Fixes and improvements
 
+## 3.0.5 of magento/product-recommendations for Magento Commerce 2.4.0
+
+-  {:.fix}Fixed "Unable to retrieve product options data" error during catalog export
```
Comment (keharper) [inline src/recommendations/release-notes.md:19]:
```suggestion
-  {:.fix}Fixed the "Unable to retrieve product options data" error that could occur during catalog export
```

File: src/recommendations/release-notes.md [hunk 2]
```diff
@@ -14,6 +14,11 @@ The release notes include:
...
+-  {:.fix}Currency symbol in **Revenue** column on the **Product Recommendations** dashboard now correctly reflects configured base currency
```
Comment (keharper) [inline src/recommendations/release-notes.md:20]:
```suggestion
-  {:.fix}The currency symbol in the **Revenue** column on the **Product Recommendations** dashboard now correctly reflects the configured base currency
```

**Comment by erikmarr** [comment]
running tests

**Comment by erikmarr** [comment]
running tests

**Pull request merged**